cmd/cronjob-checker/cronjob-checker.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: comcast.github.io/v1
+kind: KuberhealthyCheck
+metadata:
+  name: cronjob-checker
+spec:
+  runInterval: 1h
+  timeout: 10m
+  podSpec:
+    serviceAccountName: cronjob-checker
+    containers:
+      - name: cronjob-checker
+        image: kuberhealthy/cronjob-checker:v1.2.0
+        imagePullPolicy: IfNotPresent
+        env:
+          - name: NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+        resources:
+          requests:
+            cpu: 15m
+            memory: 15Mi
+          limits:
+            cpu: 25m
+        restartPolicy: Always
+    terminationGracePeriodSeconds: 5
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cronjob-checker
+rules:
+  - apiGroups:
+      - ""
+      - "events.k8s.io"
+    resources:
+      - events
+    verbs:
+      - get
+      - list
+      - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```

Comment:
should this be `ClusterRoleBinding`?

Reply:
I have moved this to a Role and Role Binding